dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsJdbcTask.java

```diff
@@ -139,11 +140,11 @@ protected TeradataLogsJdbcTask(
     this.state = Preconditions.checkNotNull(state, "SharedState was null.");
     this.logTable = logTable;
     this.queryTable = queryTable;
-    this.conditions = conditions;
+    this.conditions = ImmutableSet.copyOf(conditions);
```

Comment:
what's the benefit of using `copyOf` in here?

Reply:
same for below...

Reply:
It's a conversion from `Set` to `ImmutableSet`.